src/gsy_e/models/market/__init__.py

```diff
@@ -140,7 +140,11 @@ def orders_per_slot(self) -> Dict[str, Dict]:
         """Return all orders in the market per time slot."""
         bids = [bid.serializable_dict() for bid in self.bids.values()]
         offers = [offer.serializable_dict() for offer in self.offers.values()]
-        return {self.time_slot_str: {"bids": bids, "offers": offers}}
+        return {
```

Comment:
Do you mean updating this dict with the `MarketBase.info` dict? I can do that.

About removing the `self.time_slot_str`, this will probably have side-effects on gsy-e and myco's existing functionalities, so I'd like to keep it out of this PR for the moment.

Reply:
`About removing the self.time_slot_str, this will probably have side-effects on gsy-e and myco's existing functionalities, so I'd like to keep it out of this PR for the moment.`
Indeed, however on second look there is no need to remove it completely. You can add market_info as a separate key to the dict:
` return {self.time_slot_str: {"bids": bids, "offers": offers}, "market_info": self.info}`
Then we will be able to remove it in due time.

Reply:
Unfortunately, this would also break existing functionality. The myco expects `orders_per_slot()` to only have timeslots as keys. Adding `market_info` would then break the myco external matcher and probably some logic on the myco SDK side.
I could in theory add the `MarketBase.info` information directly inside the 2nd level dict, but this would create redundant information for the future markets:
```python
return {self.time_slot_str: {"bids": bids, "offers": offers, "market_info": self.info}
```

Reply:
This concern can be easily mitigated in 2 ways that I see on first glance though:
- (easy way to fit in the current PR but less correct): pop the "market_info" key from the `orders_per_slot` in the Myco at the time that it receives the dict, and before it starts to process it. That way, the Myco processing does not have to change at all.
- (harder way to fit in the current PR but more correct): do not add it at all in the bid-offers Redis channel, but instead add it on the "market_cycle" or the "tick" event messages that the Myco receives, and populate the market cache in these events. That way, the concerns of the 2 will not get mixed.

I would prefer the latter, but won't mind the former solution too. IMO since the duplication in the messaging can be avoided, we should try to achieve it.